tutorials/tour/_posts/2017-02-13-variances.md

```diff
@@ -10,31 +10,114 @@ next-page: upper-type-bounds
 previous-page: generic-classes
 ---
 
-Scala supports variance annotations of type parameters of [generic classes](generic-classes.html). In contrast to Java 5 (aka. [JDK 1.5](http://java.sun.com/j2se/1.5/)), variance annotations may be added when a class abstraction is defined, whereas in Java 5, variance annotations are given by clients when a class abstraction is used.
+Scala supports variance annotations of type parameters of [generic classes](generic-classes.html) similar to C#. In contrast to Java 5 (aka. JDK 1.5), variance annotations may be added when a class abstraction is defined, whereas in Java 5, variance annotations are given by clients when a class abstraction is used.
 
```

Comment:
I would suggest moving the comparison to Java and C# to a distinct section which comes at the end of the whole section, not at the beginning.